src/main/java/dev/snowdrop/release/App.java

```diff
@@ -250,12 +250,13 @@ public void setupRelease(
         } else {
             // no release ticket was specified, clone
             issue = clone(release, token);
-
         }
 
         // link CVEs
-        for (var cve : cveService.listCVEs(Optional.of(release.getVersion()), false)) {
-            service.linkIssue(issue.getKey(), cve.getKey());
+        if (!release.isTestMode()) {
+            for (var cve : cveService.listCVEs(Optional.of(release.getVersion()), false)) {
+                service.linkIssue(issue.getKey(), cve.getKey());
+            }
```

Comment:
Can we move the check on test mode into the linkIssue?

Reply:
This one I cannot change without refactoring the `linkIssue` method. I think we should leave it as is.

Reply:
ok